### ISSUE TITLE
Trident fails to boot on Dell Laptop with nVidia graphics chip

### ISSUE THREAD

**Issue by JohnBlood** [project-trident/trident-installer#28]
Last night, I attempted to install BETA3 on my Dell Lattitude D630 laptop. My laptop has an NVIDIA Quadro NVS 135M graphics chip. According to nVidia's website, that particular chip is supported by `nvidia-driver-340`. During my initial install attempt, I selected the 340 driver. The installation finished successfully. However, upon reboot, the nVidia driver appeared to fail. Below is a screenshot of the error.
![img_20180925_211851517](https://user-images.githubusercontent.com/5157847/46099804-ce495380-c195-11e8-9021-8839dbe221cd.jpg)

I attempted to reinstall with the main nVidia driver, but the same error occurred.

I successfully installed TrueOS desktop on the same laptop earlier this year.

**Comment by RodMyers** [comment]
Have you attempted to use RC3?

**Comment by JohnBlood** [comment]
No, but I will today.

**Comment by JohnBlood** [comment]
I tried RC 3 and was still unable to boot into a desktop. I tried installing without picking which driver to install, installing the nVidia 240 driver and installing the nv driver. Interestingly, when I tried to boot after installing the nv driver, it looked like it almost reached a desktop, but was tuck with a black screen.

**Comment by RodMyers** [comment]
Have you tried the RELEASE ISO?

**Comment by JohnBlood** [comment]
Not yet. I will soon.

**Comment by JohnBlood** [comment]
@RodMyers I tried installing the Project Trident and ran into issues. However, I think that the thumb drive that I was using might have contributed to the problems. I bought a new thumb drive and will again tonight. If I run into issues again, I'll post as much info as possible. :)

**Comment by JohnBlood** [comment]
I flashed Project Trident 18.12 Update 3 to a new 16GB thumb drive and tried to install but I kept getting the "zpool import failed" error. I figured that there was conflict from the previous failed installed, so I installed a Linux distro.. I reflashed Project Trident 18.12 Update 3, but the install kept failing at zpool import. Then, I decided to change the name of the zpool to trident2 and turned on swap. Once I did this, the installation completed. However, I still can't get it to boot to a gui. I tried both nvidia 340 and the open source nv driver.

**Comment by RodMyers** [comment]
is this still an issue?

**Comment by JohnBlood** [comment]
@RodMyers Sorry for not replying sooner. I am unable to test this because the laptop I was using is not working due to water damage. I will try again once I get it fixed.

**Comment by JohnBlood** [comment]
I finally got my system fixed and installed the most recent stable iso. It installed successfully, but would not boot to a desktop. I logged in via the terminal prompt and typed `startx`. The resulting printout said "Fatal server error: no screen found. Here is a picture of the screen: 
![IMG_20190909_201436021](https://user-images.githubusercontent.com/5157847/64578235-cce44d00-d34c-11e9-9ea1-d4ded7ab16e0.jpg)

Interestingly, I also ran into issues connecting to the ntp server. If I didn't log in, I kept getting a message that "Can't find host 0.freebsd.pool.ntp.org: Name does not resolve." This happened on both wifi and ethernet.

Hopefully, we can resolve this issue because I'd really like to run Project Trident on that system.
